modules/ding_library/ding_library.features.field_base.inc

```diff
@@ -224,6 +224,33 @@ function ding_library_field_default_field_bases() {
     'type' => 'image',
   );
 
+  // Exported field_base: 'field_ding_library_type'.
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-937

Reply:
This case does not describe this change.